tests/data/included/jazz/pianist/tests/functional.suite.yml

```diff
@@ -8,4 +8,4 @@
 
 class_name: TestGuy
 modules:
-    enabled: [Filesystem, TestHelper]
+    enabled: [Filesystem, \Jazz\Pianist\Codeception\Module\TestHelper]
```

Comment:
Good question! 
https://github.com/Codeception/Codeception/pull/2769/files#diff-366a0aa64f695f98df31e14f2ed8f1e0L65

There is a line that autolocating helpers by namespace is deprecated. I removed it for not to forget to do that in release.